src/main/java/jp/co/future/uroborosql/store/SqlResourceManagerImpl.java

```diff
@@ -397,7 +401,7 @@ public String getSql(final String sqlName) {
 	private void generateSqlInfos() {
```

Comment:
Could you please set the access modifier to `protected`?

This is because this logic cannot be used when building Spring/GraalVM natively and needs to be extended.

Reply:
Modifier changed to protected